### PR TITLE
coyote cli preparations

### PR DESCRIPTION
**Pull request by ViktorHy**
Added two new functions:

### CNVs as JSON
Added a parallel track to CNVs to generate JSON output compatible with coyote_cli. This included a new module for bedtools which should be configurable with external arguments in configs. As of now it does not merge the normal and tumor outputs

### COYOTE_YAML
Added a parallel track to generate yaml-output for coyote_cli. Basically a copy of the old COYOTE module but data output as yaml file to be loaded with coyote_cli yaml function

### CODE REVIEW

File: modules/local/bedtools/main.nf [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        val(intersect)
+        
+    output:
+        tuple val(group), val(meta), file("${prefix}.intersected.${suffix}"), emit: intersected
+        path "versions.yml",                                      emit: versions
+
+    script:
+        def args = task.ext.args ?: ''
+        prefix = task.ext.prefix ?: "${meta.id}.${meta.type}"
+        suffix = task.ext.suffix ?: ''
```
Comment (ramsainanduri) [inline modules/local/bedtools/main.nf:18]: It's always good to provide a default value in the script for prefix, suffix. Later they can be customized via the config.

```
suffix = task.ext.suffix ?: "bed"
```

**Comment by ramsainanduri** [comment]
> just realized I never send the cnv json over to the new coyote_yaml function

I think you are still working on it

**Comment by ViktorHy** [comment]
yeah, and I got really stuck as well. Think I have a commit coming in today

**Comment by ViktorHy** [comment]
@ramsainanduri ok now I got a working workflow for JSONs

File: modules/local/coyote/main.nf
Comment (ramsainanduri) [inline modules/local/coyote/main.nf:1]: coyote yaml is not published, it needs to be added in the add_to_db.config

**Review by ramsainanduri — APPROVED** [review]
LGTM

**Pull request merged**